package.json

```diff
@@ -4,7 +4,8 @@
   "description": "SAFE Browser add-on for web client to access SAFE Network",
   "main": "index.js",
   "scripts": {
-    "lint": "eslint src *.js",
+    "fix-lint": "eslint --fix src",
+    "lint": "eslint src",
```

Comment:
Hey, @hunterlester just a simple improvement. can we change this line to be `"lint": "eslint --cache --format=node_modules/eslint-formatter-pretty .",` with `eslint-formatter-pretty` dev-dependency

Reply:
Should I include `.eslintcache` in commit? Or ignore it?